Pointer error on status command when bootstrap URL is not set

```shell
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13efd7d]

goroutine 1 [running]:
github.com/bf2fc6cc711aee1a0c2a/cli/pkg/cmd/kafka/status.runStatus(0xc000427710, 0xc0004af220, 0x0)
        /home/ephelan/code/github.com/bf2/cli/pkg/cmd/kafka/status/status.go:88 +0x51d
github.com/bf2fc6cc711aee1a0c2a/cli/pkg/cmd/kafka/status.NewStatusCommand.func1(0xc0004b02c0, 0xc00046d1d0, 0x0, 0x1, 0x0, 0x0)
        /home/ephelan/code/github.com/bf2/cli/pkg/cmd/kafka/status/status.go:53 +0xdc
github.com/spf13/cobra.(*Command).execute(0xc0004b02c0, 0xc00046d1c0, 0x1, 0x1, 0xc0004b02c0, 0xc00046d1c0)
        /home/ephelan/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000416840, 0x19443f0, 0x11, 0xc000416840)
        /home/ephelan/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/ephelan/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
        /home/ephelan/code/github.com/bf2/cli/cmd/rhoas/main.go:38 +0xe5
```

--- a/pkg/cmd/kafka/status/status.go
+++ b/pkg/cmd/kafka/status/status.go
@@ -82,10 +82,10 @@ func runStatus(opts *Options) error {
 	}
 
 	statusInfo := &kafkaStatus{
-		ID:            *res.Id,
-		Name:          *res.Name,
-		Status:        *res.Status,
-		BootstrapHost: *res.BootstrapServerHost,
+		ID:            res.GetId(),
+		Name:          res.GetName(),
+		Status:        res.GetStatus(),
+		BootstrapHost: res.GetBootstrapServerHost(),
 	}
 
 	dump.Table(os.Stdout, statusInfo)

